src/BaseBallGame.js

```diff
@@ -0,0 +1,44 @@
+export default class BaseballGame {
+  play(computerInputNumbers, userInputNumbers){
+    let strikeCount = 0;
+    let ballCount = 0;
+
+    for(let i = 0 ; i < 3; i+=1){
+      let result = this.compare(computerInputNumbers,userInputNumbers[i], i);
+      if(result === 'strike'){
+        strikeCount+=1;
+      }
+      else if(result === 'ball'){
+        ballCount+=1;
+      }
+    }
+    return this.makeResult(strikeCount,ballCount);
+  }
+
+  compare(computerNumbers, userNumber, idx){
+    for(let i =0; i < computerNumbers.length ; i+=1){
+      if(computerNumbers[i] === userNumber && i === idx){
+        return 'strike';
+      }
+      if(computerNumbers[i] === userNumber && i !== idx){
+        return 'ball';
+      }
+    }
+    return 'Nothing';
+  }
+
+  makeResult(strikeCount, ballCount){
+    let output = '';
+    if(ballCount !== 0){
+      output += (ballCount.toString() + '볼 '); 
+    }
+    if(strikeCount !==0){
+      output += (strikeCount.toString() +'스트라이크')
+    }
```

Comment:
템플릿 리터럴을 사용하는 것은 어떨까요?
```suggestion
    if(ballCount !== 0){
      output += `${ballCount}볼 `;
    }
    if(strikeCount !==0){
      output += `${ballCount}스트라이크`;
    }
```

Reply:
이런 기능이 있는지 몰랐습니다! 다음부터 문자열을 생성할 때는 템플릿 리터럴를 사용해서 작성해 보겠습니다! 감사합니다.